### PR TITLE
Fixed areURLReferentsEqual; PluginManager dialog tuning

### PR DESCRIPTION
**Pull request by mpyat2**
Fixed areURLReferentsEqual; in the PluginManager dialog, an item is grayed if local and remote plug-ins differ or if the remote plug-in does not exist.

### CODE REVIEW

**Comment by mpyat2** [comment]
Issue #373 fix

File: src/org/aavso/tools/vstar/ui/dialog/plugin/manager/PluginManager.java [hunk 1]
```diff
@@ -983,19 +983,16 @@ private List<Byte> getBytesFromURL(URL url) throws IOException {
 		List<Byte> byteList = new ArrayList<Byte>();
 		InputStream stream = url.openStream();
 		try {
-			int len = stream.available();
-			while (len > 0) {
-				byte[] bytes = new byte[len];
-				stream.read(bytes, 0, len);
-				for (byte b : bytes) {
-					byteList.add(b);
+			byte[] buf = new byte[4096];
```
Comment (dbenn) [inline src/org/aavso/tools/vstar/ui/dialog/plugin/manager/PluginManager.java:986]: Do we need to be careful about the use of 4096 here @mpyat2 ?

Comment (mpyat2) [reply src/org/aavso/tools/vstar/ui/dialog/plugin/manager/PluginManager.java:986]: Hi @dbenn , well, actually, this is a somewhat modified 'copy' method (see line 945) that has worked for years. What could be wrong with a 4096 buffer?

Comment (dbenn) [reply src/org/aavso/tools/vstar/ui/dialog/plugin/manager/PluginManager.java:986]: @mpyat2, you're right of course. I'm thinking like a C programmer! Progress on the road to senility? :) I'm happy from https://docs.oracle.com/javase/8/docs/api/java/io/InputStream.html#read-byte:A- that "The number of bytes read is, at most, equal to the length of b."

**Review by dbenn — APPROVED** [review]
Looks good. Thanks @mpyat2!

**Pull request merged**